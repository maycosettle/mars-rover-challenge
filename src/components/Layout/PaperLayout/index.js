export { default as PaperLayout } from './PaperLayout';
