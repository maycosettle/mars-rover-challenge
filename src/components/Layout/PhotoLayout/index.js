export { default as PhotoLayout } from './PhotoLayout';
