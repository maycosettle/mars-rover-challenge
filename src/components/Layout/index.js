export * from './PageLayout';
export * from './PaperLayout';
export * from './PhotoLayout';
