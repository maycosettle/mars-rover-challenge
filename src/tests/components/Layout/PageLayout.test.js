import { shallow } from 'enzyme';
import { Header } from 'components/Layout';

describe('tests on <Header />', () => {
  const wrapper = shallow(<Header />);

  test('Should render Header correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
