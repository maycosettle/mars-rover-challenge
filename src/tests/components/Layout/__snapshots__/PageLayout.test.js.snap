// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <Header /> Should render Header correctly 1`] = `
<div
  className="pt-4 sm:py-4 sm:flex sm:flex-col sm:items-center sm:px-6 lg:px-8"
>
  <div
    className="w-full flex-col sm:flex-row sm:pb-4"
  >
    <span
      className="font-display sm:text-2xl flex items-center justify-center font-semibold text-primary-500 text-xl"
    />
  </div>
  <Divider
    className=""
  />
</div>
`;
